api/src/main/resources/workbench.yaml

```diff
@@ -213,6 +213,43 @@ paths:
           schema:
             $ref: "#/definitions/Profile"
 
+  /api/v1/create-registered-group:
```

Comment:
I don't think we want to add create-controlled-group, etc. later when we add the controlled authorization domain.

I also think if you have admin tools that let you remove somebody from the group, they probably should allow adding them back in.

Probably instead we could have something like Firecloud:

/api/v1/groups:

  post:
    (creates an authorization domain -- request specifies the name of the group, which we can optionally validate matches a whitelist)

/api/v1/groups/<group name>/users/<user ID>

   post:
    adds a user to a group

  delete:
    deletes a user from a group

WDYT?